public/js/projectView.js

```diff
@@ -2,16 +2,16 @@
 
 var app = app || {};
 
-
-app.makeSectionsAppear = function() {
-  $('.topNav').on('click', '.navTab', function(event) {
-    event.preventDefault();
-    $('.sectionContent').hide();
-    $('#' + $(this).attr('data-content')).fadeIn();
-  });
-
-  $('.topNav .navTab:first').click();
-};
+//
+// app.makeSectionsAppear = function() {
+//   $('.topNav').on('click', '.navTab', function(event) {
+//     event.preventDefault();
+//     $('.sectionContent').hide();
+//     $('#' + $(this).attr('data-content')).fadeIn();
+//   });
+//
+//   $('.topNav .navTab:first').click();
+// };
 
 
 $(document).ready(function() {
```

Comment:
Make sure you remove this as well since `makeSectionsAppear` no longer is defined.

Reply:
removed.